extensions/wikia/HAWelcome/HAWelcomeHooks.php

```diff
@@ -23,186 +23,6 @@ public static function queueHAWelcomeTask( $wgCityId, $oTitle, $aParams ) {
 	}
 
 	/**
```

Comment:
Looks right. This was the previous version before the port to the new job queue.